pyk/src/tests/integration/test_krun_proof_hints.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pyk.kllvm.hints.prooftrace as prooftrace
+from pyk.kore.parser import KoreParser
+from pyk.testing import KRunTest
+
+if TYPE_CHECKING:
+    from pyk.ktool.krun import KRun
+
+
+class Test0Decrement(KRunTest):
```

Comment:
Consider

```suggestion
class Test0Decrement(ProofTraceTest, KRunTest):
```

Then all the new additions to `_kompiler.py` become unnecessary I think.

Reply:
Good point! Thanks!